azds.yaml

```diff
@@ -0,0 +1,40 @@
+kind: helm-release
+apiVersion: 1.0
+build:
+  context: .
+  dockerfile: Dockerfile
+install:
+  chart: charts/mywebapi
+  values:
+  - values.dev.yaml?
+  - secrets.dev.yaml?
+  set:
+    image:
+      tag: $(tag)
+      pullPolicy: Never
+    disableProbes: true
+    # This ingress section specifies whether AZDS should expose
```

Comment:
The service should be public by default, using AKS built-in ingress

Reply:
Agreed! I'll make a change and commit

Reply:
Latest change look good!